core/pom.xml

```diff
@@ -66,11 +66,23 @@ THE SOFTWARE.
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>remoting</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>org.jenkins-ci</groupId>
+          <artifactId>constant-pool-scanner</artifactId>
+        </exclusion>
+      </exclusions>
     </dependency>
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>cli</artifactId>
       <version>${project.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>net.i2p.crypto</groupId>
+          <artifactId>eddsa</artifactId>
```

Comment:
It is a pluggable library for encryption, which needs to be in classpath to support EdDSA signature. I would be excludin it here would have no effect since there are other dependencies, but I do not see particular point in it. 

Rework of https://github.com/jenkinsci/jenkins/pull/4110 would be a better option (see the discussion)

Reply:
I see your point. Thanks :+1: